.github/workflows/pr-checks.yaml

```
@@ -77,7 +77,7 @@ jobs:
 
           make manifests
           git update-index -q --refresh
-          if ! git diff-index --quiet HEAD -- config/ ; then
+          if ! git diff-index --quiet HEAD -- config/ pkg/primaza/workercluster/templates/ ; then
```

Can maybe take this out, since that directory is covered by a `.gitignore`.

```suggestion
          if ! git diff-index --quiet HEAD -- config/
```

---

done!